Python/sysmodule.c

```diff
@@ -205,12 +199,32 @@ sys_audit_tstate(PyThreadState *ts, const char *event,
 
     /* Early exit when no hooks are registered */
     PyInterpreterState *is = ts->interp;
-    if (!should_audit(is)) {
+    if (!should_audit_interp(is)) {
         return 0;
     }
+    return 1;
```

Comment:
Why not simply
```c
return should_audit_interp(ts->interp);
```
?

Why not simply inline it as `ts && should_audit_interp(ts->interp)`?

Reply:
I reverted my other sysmodule.c to make this PR as small as possible. Previous changes were needed when PySys_AuditTuple() didn't reuse PySys_Audit().